docs/porting/connectivity/Layer3IP.md

```diff
@@ -0,0 +1,142 @@
+<h2 id="ethernet-port">IP Layer3 (L3IP) drivers</h2>
+
+<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/l3ip.png)<span>L3IP API</span></span>
+
+This document describes how to port and test a cellular IP Layer3 (L3IP) driver to Mbed OS.
+
+The scope of this document is limited to advanced cellular modems, where the device presents an additional IP Layer3  interface to send and receive frames, and one of the onboard network stacks that runs on Mbed OS on the host processor uses this.
+
+If the device has an off-board network stack, a driver needs to implement
+ [NetworkStack](https://github.com/ARMmbed//mbed-os-5-docs/blob/development/docs/porting/connectivity/NetworkStack.md)   directly instead to pass network calls to the offboard stack.
```

Comment:
This link needs to be fixed.
I think the correct target is https://os.mbed.com/docs/mbed-os/v5.11/porting/networkstack.html

Which is (networkstack.html), or sometimes `../porting/networkstack.html` have been used.